lib/config/index.js

```diff
@@ -95,7 +95,15 @@ config.isLDAPEnable = config.ldap.url
 config.isPDFExportEnable = config.allowpdfexport
 
 // generate correct path
-config.sslcapath = path.join(appRootPath, config.sslcapath)
+config.sslcapath = (function () {
+  var i, len, results
+  results = []
+  for (i = 0, len = config.sslcapath.length; i < len; i++) {
+    results.push(path.join(appRootPath, config.sslcapath[i]))
```

Comment:
Please use [`path.resolve()`](https://nodejs.org/api/path.html#path_path_resolve_paths) instead of path.join.

Reply:
Why don't we replace other path.join() in index.js into path.resolve()?

Reply:
We'll do this while refactoring, but right now we touch this part of the code and we should use `resolve()` since it doesn't break absolute paths. :)

Reply:
Why we need to use resolve() instead of join()?As I know they seems do the same thing there.

Reply:
While join will simply append the added path, resolve does always return an absolute path to the file. And it can handle absolute path as input, which join doesn't.

so when you put `path.join("/opt/hackmd", "/tmp/somefile")` you'll get `/opt/hackmd/tmp/somefile` while `path.resolve("/opt/hackmd", "/tmp/somefile")` returns `/tmp/somefile`.

Reply:
oh well.I got it.I just wondered why we'll need put two absolute path as parameter,that really weird since it isn't necessary.
Because if I specified an absolute path in config,the join() can't handle it as expected,right?:)

Reply:
`resolve` takes absolute paths as well as relatives, but always returns an absolute one.

As mentioned before: `path.resolve("/opt/hackmd", "/tmp/somefile")` returns `/tmp/somefile`.
But also possible: `path.resolve("/opt/hackmd", "config/somefile")` returns `/opt/hackmd/config/somefile`.

So `resolve` does both, while `join` can only handle relative paths.